.prettierrc.json

```diff
@@ -1,4 +1,4 @@
 {
   "singleQuote": true,
-  "endOfLine": "auto"
+  "endOfLine": "lf"
```

Comment:
actually "lf" is the default, so we can entirely remove this line

Reply:
P.S. I'm primarily setting this because CRLF is leaking into the repo, and Windows can usually handle LF these days. You might prefer to fix your Git setting instead of changing though, if something breaks. I think it means that you have `core.autocrlf` set to `false`, while it should be `true` (on Windows).